policykit/constitution/models.py

```diff
@@ -247,6 +247,25 @@ class EditorModel(ConstitutionAction):
     class Meta:
         abstract = True
 
+    def get_existing_policy_variables(self):
+        if self.variables:
+            return [PolicyVariable.objects.get(pk=id) for id in self.variables.keys()]
```

Comment:
`get` will raise an error if any of the `PolicyVariable`s have since been deleted. I think we want to ignore changes to variables that have been deleted. Something like this (untested)
```suggestion
            vars = [PolicyVariable.objects.filter(pk=id).first() for id in self.variables.keys()]
            return [var for var in vars if var is not None]
```

Reply:
Thank you, I updated this line!